plugins/hls-eval-plugin/test/Main.hs

```diff
@@ -84,8 +84,8 @@ tests =
       evalInFile "T8.hs" "-- >>> 3 `div` 0" "-- divide by zero" -- The default for marking exceptions is False
   , goldenWithEval "Applies file LANGUAGE extensions" "T9" "hs"
   , goldenWithEval "Evaluate a type with :kind!" "T10" "hs"
-  , goldenWithEval' "Reports an error for an incorrect type with :kind!" "T11" "hs"
-        (if ghcVersion >= GHC94 then "ghc94.expected" else "expected")
+  -- TODO: I do not understand why this is 'ghc94.expected'.
```

Comment:
Why `ghc94.expected`?

Reply:
I presume this is a result of code churn, initially the test was only failing for GHC94, then we added support for GHC96... Would be fine to just use `goldenWithEval`.